manuals/1.0/en/bp/DontReuseAnnotations.md

```diff
@@ -1,35 +1,34 @@
 ---
 layout: docs-en
-title: Don't reuse binding annotations
+title: DontReuseAttributes
 category: Manual
 permalink: /manuals/1.0/en/bp/dont_reuse_annotations.html
 ---
-# Don't reuse binding annotations (aka `@Qualifiers`)
+# Don't reuse binding attribute (aka `#[Attribute]`)
 
-Sometimes, of course, it makes sense to bind some highly-related bindings with
-the same annotations. E.g. `@ServerName String` and `@ServerName CharSequence`.
+Sometimes, of course, it makes sense to bind some highly-related bindings with the same attributes. E.g. `#[ServerName]`
 
-That said, most binding annotations should only qualify one binding. And you
-should definitely not reuse a binding annotation for *unrelated* bindings.
+That said, most binding attributes should only qualify one binding. And you should definitely not reuse a binding attributes for *unrelated* bindings.
 
-When in doubt, don't reuse annotations: creating one is straightfoward!
+When in doubt, don't reuse attributes: creating one is straightfoward!
 
-To avoid some boilerplate, sometimes it makes sense to use annotation parameters
-to create distinct annotation instances from a single declaration. For example:
+To avoid some boilerplate, sometimes it makes sense to use attribute parameters to create distinct annotation instances from a single declaration. For example:
 
-```java
-enum Thing { FOO, BAR, BAZ }
+```php
+enum Thing
+{
+    case FOO;
+    case BAR;
+    case BAZ;
+}
 
-@Qualifier
-@Retention(RetentionPolicy.RUNTIME)
-@interface MyThing {
-  Thing value();
+#[Attribute, \Ray\Di\Di\Qualifier]
+final class MyThing
+{
+    public function __construct(
+        public readonly Thing $value
+    ) {}
 }
 ```
 
-You can then use `@MyThing(FOO)`, `@MyThing(BAR)`, and `@MyThing(BAZ)` rather
-than defining each of them as separate annotation types.
-
-To construct `Annotation` object instances for parameterized annotations (e.g.
-to use in constructing a `Key` object), you can use the
-[auto annotation helper](https://github.com/google/auto/blob/master/value/userguide/howto.md#-use-autovalue-to-implement-an-annotation-type).
+You can then use `#[MyThing(Thing::FOO)]`, `#[MyThing(Thing::BAR)]`, and `#[MyThing(Thing::BAZ)]` rather than defining each of them as separate annotation types.
```

Comment:
```suggestion
You can then use `#[MyThing(Thing::FOO)]`, `#[MyThing(Thing::BAR)]`, and `#[MyThing(Thing::BAZ)]` rather than defining each of them as separate attribute types.
```